website/www/site/content/en/blog/2016-02-22-beam-has-a-logo.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title:  "Apache Beam has a logo!"
+slug: "beam-has-a-logo.html"
+date:   2016-02-22 10:21:48 -0800
+excerpt_separator: <!--more-->
+categories: beam update website
+authors:
+- jamesmalone
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+One of the major benefits of Apache Beam is the fact that it unifies both
+both batch and stream processing into one powerful model. In fact, this unification
+is so important, the name Beam itself comes from the union of **B**atch + str**EAM** = Beam
+
+When the project started, we wanted a logo which was both appealing and visually
+represented this unification. <!--more--> Thanks to the **amazing** work of Stephanie Smythies, the Apache Beam project
```

Comment:
Oh it seems that this `<!--more-->` breaks into the new line. 🤔

Reply:
I know, but it seems that that's how Hugo parses it :( I did some research and people were complaining about it, but there's no solution yet. It simply wraps the summary in `<p>` and that's why line breaks